tests/plugins/emoji/manual/dropdown/scrollpagebug.md

```diff
@@ -1,11 +1,11 @@
-@bender-tags: 4.11.1, bug, emoji, 2571
+@bender-tags: 4.11.1, bug, emoji, 2571, 4.11.3, 2592
 @bender-ckeditor-plugins: wysiwygarea, toolbar, emoji
 @bender-ui: collapsed
 @bender-include: ../../_helpers/tools.js
 
 1. Open emoji dropdown.
 2. Click into emoji group.
-### Expected
-Page is not scrolled down.
 ### Unexpected
-Page scrolls down.
+Page is not scrolled down.
+### Expected
```

Comment:
Actually with your change the manual test has first unexpected value and then expected. That's an uncommon notation in manual tests.

I'll flip the order during the review.